components/admin/lessons/AdminLessonNav.tsx

```diff
@@ -0,0 +1,74 @@
+import { toUpper } from 'lodash'
+import React, { useState } from 'react'
+import { Tab, Col, Nav } from 'react-bootstrap'
+import styles from '../../../scss/adminLessonNav.module.scss'
+
+type AdminLessonNavProps = {
+  navItems: { value: string }[]
```

Comment:
Can a person pass in an empty array with this type definition?

Reply:
They can't I think and they shouldn't as it'll break it. There will be tabs but no navItems to use for switching between tabs.

Reply:
What i'm worried about is that on line 54 we do `tabs[index]()`. If someone is able to pass in an empty array for `tabs`, then `tabs[index]()` will break things. Can we confirm that the type for `tabs` blocks people from being able to pass in an empty array?
https://github.com/garageScript/c0d3-app/pull/1637/files#diff-d8433cac2e9017e63fddfdb72a729f8fe01dcd7945242e577aea97802e299c6dR54